os-partition/added/partitionPV.sh

```diff
@@ -48,6 +50,10 @@ function partitionPV() {
           SERVER_DATA_DIR="${INSTANCE_DIR}/serverData"
           mkdir -p "${SERVER_DATA_DIR}"
 
+          if [ -d "$S2I_DATA_DIR" ]; then
+            cp -rf ${S2I_DATA_DIR}/* ${SERVER_DATA_DIR}
+          fi
+ 
```

Comment:
This should only run on the first instance, i.e. when we create the directory for the first time.  It should also be handled through a function, perhaps init_server_data or similar, so there are no product specific details within this script.  We would need to stub this out in the A-MQ script since I don't believe it has requirements for initialising the data dir.